data/Hrishi5111998.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Hrishikesh Tule",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Web developer **Digital Nomad**. open source enthusiast",
+  "avatar": "https://github.com/Hrishi5111998.png",
+  "links":[
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/@hrishikesh7030",
+      "icon": "FaTwitter"
+    },
+    {
+     "name": "Instagram",
+     "url": "https://instagram.com/@hrishikesh2412",
+     "icon": "FaInstagram"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://linkedin.com/in/hrishikesh-tule",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/Hrish5111998",
```

```suggestion
      "url": "https://github.com/Hrishi5111998",
```
I believe you misspelled your username. please update.

---

Great catch @ChinmayMhatre, love seeing your contribution to lots of PRs! 😍

---

> 

oh my bad I will update

---

Thank you @kumarsonsoff3🙌. Great seeing you too.